oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -1,15 +1,12 @@
 package com.google.oak.functions.client;
 
 import com.google.common.base.VerifyException;
-import com.google.oak.functions.client.AeadEncryptor;
-import com.google.oak.functions.client.KeyNegotiator;
```

Comment:
Why are these lines removed? Can Java see classes from the same package by default?

Reply:
Yes. So these imports are redundant.